blog/modular-models-announcement.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Modular Models
+description: Modular Models 
+slug: modular-models-announcement
+date: 2024-04-21
+authors: eharris
+tags: [openfga,features]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Release Candidate for Modular Models
+
+Modular models aims to improve the model authoring experience when multiple teams are maintaining a model, such as:
+
+- A model can grow large and difficult to understand
+- As more teams begin to contribute to a model, the ownership boundaries may not be clear and code review processes might not scale
+
+With modular models, a single model can be separated across multiple files allow grouping of types and conditions into modules. This means a model can more easily be organized in terms of team or organizational structure and by using a feature such as [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/codeowners/) code owners, it should become easier to ensure the owners of an portion of your model are correctly assigned to review it.
+
+## How to use it?
+
+Modular models is currently shipped as experimental while we gather feedback, in order to try it out.
+
+* Download the [beta release](https://github.com/openfga/cli/releases/tag/v0.3.0-beta.1) of the CLI
+  * With the CLI you will be able to write your modular model and test it against a store file you have, but you will not be able to write your model to any OpenFGA server that does not enable this feature.
+* Download [v1.5.1](https://github.com/openfga/openfga/releases/tag/v1.5.1) of OpenFGA
+  * As this is currently in RC you will need to start OpenFGA with `openfga run --experimental enable-modular-models`.
+* Check out the modular models sample store in the [sample-stores repo](https://github.com/openfga/sample-stores/tree/main/stores/modular)
+* The latest version of VS Code contains support for modular models, so ensure you're updated
+* Review the [documentation for this feature](/docs/modeling/modular-models)
+
+## What's next?
+
+We intend to move this feature out of RC within the next few weeks, so as we gather feedback we'll deal with any issues that arise and look to improve upon any areas of the developer experience as needed.
+
+Looking beyond the near term, modular models allows us to start evaluating other features such as per-module authorization (a.k.a FGA-on-FGA) that would allow providing restrictions on what applications can write tuples for or run queries against.
```

Comment:
Is FGA-on-FGA something that's widely understood outside of the team? I wonder if it would be less confusing not to mention it.

Reply:
It's been mentioned in a few places, but I'm not sure if we have a discussion or roadmap item to reference. @rhamzeh do you know of anything that could be linked to here?